### DISCUSSION TITLE
Does validation data impact the model performance?

### DISCUSSION THREAD

**Discussion by felix-knopp** [ultralytics/yolov5#6023] [Q&A] [answered]
Is validation data only used to measure performance on data separate from the training data or does it have any effect on the training results?

If it is only for measuring performance during training I would choose a small validation set and increase the training and testing set.

**Comment by glenn-jocher** [comment, marked as answer]
@felix-knopp validation set is separate from train set and is used to measure mAP during/after training.
https://github.com/ultralytics/yolov5/blob/abbdd4802ea1f01c9e8c723d3792e0e0a92d604e/data/coco.yaml#L10-L15

**Reply by felix-knopp** [reply]
Ok. So I assume that the validation set has no impact on the resulting model at all. 

It's important to me because i'm training with computer generated images and want to compare the results with a training on "real" data .
I'm using the same validation set (containing "real" data) for both trainings and want to make sure, that the results on the CG train set are not impacted by using "real" data as validation.

**Reply by glenn-jocher** [reply]
@felix-knopp yes model weights are not updated based on validations set, only training set.

**Reply by pretiand** [reply]
so, can we say that the validation set is used only to compute metrics every epochs? Or does it adjust the training in someway, (i.e. influencing the learning rate or other hyperparameters)?

**Reply by glenn-jocher** [reply]
val set is only used to compute metrics
